libraries/ESP8266WiFi/src/WiFiClientSecureBearSSL.cpp

```diff
@@ -100,20 +100,15 @@ void WiFiClientSecure::_clearAuthenticationSettings() {
 }
 
 
-WiFiClientSecure::WiFiClientSecure() : WiFiClient() {
+WiFiClientSecureCtx::WiFiClientSecureCtx() : WiFiClient() {
   _clear();
   _clearAuthenticationSettings();
   _certStore = nullptr; // Don't want to remove cert store on a clear, should be long lived
   _sk = nullptr;
   stack_thunk_add_ref();
 }
 
-WiFiClientSecure::WiFiClientSecure(const WiFiClientSecure &rhs) : WiFiClient(rhs) {
-  *this = rhs;
-  stack_thunk_add_ref();
```

Comment:
Did we lose an increment to the stack thunk refcnt here?  Or was the refcnt increment in error in the original code?

Reply:
No. The copy constructor [is now disabled](https://github.com/esp8266/Arduino/pull/7680#discussion_r513378742).
That is in fact the purpose of this PR.